docker/config/prometheus/prometheus.yml

```diff
@@ -8,4 +8,4 @@ scrape_configs:
   - job_name: lido_terra
     scrape_interval: 1m
     static_configs:
-    - targets: ['lido_terra:8080']
+    - targets: ['lido_terra:8080', 'governance_bot:8080']
```

Comment:
does it work with current setup? i mean `governance_bot` in a different docker network on the prod server, we are unable to get it with a hostname. Seems like we either need to find a way to configure a targets with env variables or move `governance_bot` to the `monitoring` docker-compose to simplify avaliability via network

Reply:
We can run governance bot on a same network with `network=x`.
So I thought it will work for now.
There will be another problem though, when we move governance bot on another machine. (probably can be solved with overlay network in docker)